JS/Lesson22/tugba/index.html

```diff
@@ -1,11 +1,30 @@
+
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Accordion</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Accordion</title>
 </head>
 <body>
-  
+    <button class="accordion">Section 1</button>
```

Comment:
I'd recommend to change HTML structure like this for easier styling in CSS:
```
<div class="accordion__container">
			<button class="accordion">Section 1</button>
			<div class="panel active">
				<p>
					Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
					eiusmod tempor incididunt ut labore et dolore magna aliqua.
				</p>
			</div>
		</div>
		<div class="accordion__container">
			<button class="accordion">Section 2</button>
			<div class="panel active">
				<p>
					Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
					nisi ut aliquip ex ea commodo consequat.
				</p>
			</div>
		</div>
		<div class="accordion__container">
			<button class="accordion">Section 3</button>
			<div class="panel active">
				<p>
					Duis aute irure dolor in reprehenderit in voluptate velit esse cillum
					dolore eu fugiat nulla pariatur.
				</p>
			</div>
		</div>
```

This way every accordion element is in it's own div container so you can apply flexbox / grid and slign items easier.